app/api/helpers/permission_manager.py

```diff
@@ -94,7 +109,8 @@ def is_coorganizer_endpoint_related_to_event(view, view_args, view_kwargs, *args
         _jwt_required(app.config['JWT_DEFAULT_REALM'])
         return view(*view_args, **view_kwargs)
 
-    if user.is_organizer(kwargs['event_id']) or user.is_coorganizer(kwargs['event_id']):
+    if user.is_owner(kwargs['event_id']) or user.is_organizer(kwargs['event_id']) or \
+            user.is_coorganizer(kwargs['event_id']):
```

Comment:
These 3 conditions are very common and repeated. Move to user model in a function and repeat everywhere